src/components/network/NetworkView.spec.tsx

```diff
@@ -36,9 +36,11 @@ describe('NetworkView Component', () => {
     id: string | undefined,
     status?: Status,
     images?: string[],
+    externalNetworkName?: string,
     withBitcoinData = true,
   ) => {
     const network = getNetwork(1, 'test network', status);
+    externalNetworkName ? (network.externalNetworkName = externalNetworkName) : null;
```

Comment:
This is a whacky looking line of code. This would be easier to read:
```ts
const network = {
  ...getNetwork(1, 'test network', status),
  externalNetworkName,
};
```